docusaurus/docs/dev-docs/configurations/features.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Features configuration
+sidebar_label: Features
+description: Enable experimental Strapi features
+displayed_sidebar: devDocsConfigSidebar
+---
+
+# Features configuration
+
+The `config/features.js|ts` file is used to enable feature flags. Currently this file only includes a `future` object used to enable experimental features through **future flags**. 
+
+Some incoming Strapi features are not yet ready to be shipped to all users, but we want to offer community users the opportunity to provide early feedback on these new features or changes.
+
+Such experimental features are indicated by a <FutureBadge /> badge throughout the documentation and enabling these features requires enabling the corresponding future flags.
+
+:::danger
+Enable future flags at your own risk. Experimental features may be subject to change or removal, may contain breaking changes, may be unstable or not fully ready for use, and some parts may still be under development or using mock data.
+:::
+
+<!-- ! Commented out as not relevant for now -->
+<!-- Future flags can also be utilized for enabling coming breaking changes in upcoming versions (when prefixed by `vX`, with 'X' being the target version). In this scenario, if you decide to enable a future flag for a breaking change, you will need to migrate your application to adapt to this breaking change. -->
+
```

Comment:
Do you think it's worth adding something like:

> Once an unstable feature reaches a stable state, we remove the special prefix and include the feature in the next minor release. At this point, the API's structure remains consistent throughout subsequent minor releases.

I think it wraps up how you've described the development process nicely.

Reply:
Totally worth it. I'll include it as-is (though this is in a comment for now, we'll uncomment it when necessary).